server/models/schema.sql

```diff
@@ -1978,6 +1978,17 @@ CREATE TABLE `stock_requisition_item` (
   CONSTRAINT `stock_req_item__stock_req_item` FOREIGN KEY (`requisition_uuid`) REFERENCES `stock_requisition` (`uuid`)
 ) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
 
+DROP TABLE IF EXISTS `stock_adjustment_log`;
+CREATE TABLE `stock_adjustment_log` (
+  `document_uuid` BINARY(16) NOT NULL,
+  `log_timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Is there a particular reason you'd want to call this `log_timestamp`?  If not, let's call it the usual `created_at`, like we have in a couple other tables here.